ohsome_quality_analyst/api/request_models.py

```diff
@@ -29,7 +50,7 @@ class IndicatorRequest(BaseBpolys):
         ...,
         title="Topic Key",
         alias="topic",
-        example="building-count",
+        # example="building-count",
```

Comment:
Why is this commented out and not removed? If you need it later again, you could add a TODO.

Reply:
```suggestion
```

Reply:
removed comment